.pre-commit-config.yaml

```diff
@@ -88,5 +88,4 @@ repos:
           - types-requests
           - types-tqdm
           - types-flask
-          - pytket-qiskit
-          - qiskit[all]
+          - pytket_qiskit
```

Comment:
This is missing the extra Qiskit packages, right?
Although they are probably not typed either, it would be more correct to add them here since they are a requirement of mqt-bench.

Reply:
True. I added them, but only the three necessary ones since the `all` will be deprecated soon if I understood correctly.

Reply:
`[all]` won't be deprecated, but it is significantly reduced and does not contain all the application modules any more.

Reply:
But I guess it is still preferred to exactly name the necessary modules? Or would you argue in favor of `all`?

Reply:
Nope, no visualizations so far.

Reply:
In that case, feel free to resolve this.